bootstrap-cmake.sh

```diff
@@ -23,8 +23,8 @@ fi
 cmake3 \
     -DCMAKE_INSTALL_PREFIX=/opt/vespa \
     -DJAVA_HOME=/usr/lib/jvm/java-openjdk \
-    -DEXTRA_LINK_DIRECTORY="/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib64/llvm3.9/lib" \
-    -DEXTRA_INCLUDE_DIRECTORY="/opt/vespa-boost/include;/opt/vespa-cppunit/include;/usr/include/llvm3.9" \
-    -DCMAKE_INSTALL_RPATH="/opt/vespa/lib64;/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib/jvm/java-1.8.0/jre/lib/amd64/server;/usr/lib64/llvm3.9/lib" \
+    -DEXTRA_LINK_DIRECTORY="/opt/vespa-gtest/lib;/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib64/llvm3.9/lib" \
+    -DEXTRA_INCLUDE_DIRECTORY="/opt/vespa-gtest/lib;/opt/vespa-boost/include;/opt/vespa-cppunit/include;/usr/include/llvm3.9" \
```

Comment:
Should this be `/opt/vespa-gtest/include` and not `lib`?

Reply:
Good catch. Fixed in new commit.